src/pages/Validators/Index.tsx

```diff
@@ -8,6 +10,15 @@ import {ValidatorsTable} from "./ValidatorsTable";
 
 export default function ValidatorsPage() {
   const [state, _] = useGlobalState();
+  const [onDelegatory, setOnDelegatory] = useState<boolean>(false);
+  const inDev = useGetInDevMode();
+
+  const handleTabChange = (
+    _event: React.SyntheticEvent,
```

Comment:
is the `_event` needed?

Reply:
yes, event itself is not used in this `handleTabChange` function, but it's needed in `<Tabs onChange={xx}>` cuz `event` is: "The event source of the callback" to be consumed by `onChange`